jit/analyze.ts

```diff
@@ -65,6 +65,7 @@ module J2ME {
     "java/lang/Thread.yield.()V": YieldReason.Root,
     "java/lang/Thread.start0.()V": YieldReason.Root,
     "java/lang/Class.forName0.(Ljava/lang/String;)V": YieldReason.Root,
+    "java/lang/Class.newInstance1.(Ljava/lang/Object;)V": YieldReason.Root,
```

Comment:
We should also remove "java/lang/Class.newInstance.()Ljava/lang/Object;" from this list, right?